packages/deck-scripts/config/baseModuleRollupConfig.js

```diff
@@ -0,0 +1,27 @@
+const baseRollupConfig = require('./baseRollupConfig');
+const angularJsTemplateLoader = require('../helpers/rollup-plugin-angularjs-template-loader');
+
+module.exports = {
+  ...baseRollupConfig,
+  output: [{ dir: 'dist', format: 'es', sourcemap: true }],
+  plugins: [...baseRollupConfig.plugins, angularJsTemplateLoader({ sourceMap: true })],
+  external: [
```

Comment:
I'm curious how you came to this list of externals?

Reply:
I usually lean towards putting all the dependencies as externals but I don't know if it's better :)

Reply:
I should make a comment here explaining that this will be updated as I migrate more spinnaker modules. I came up with this list by looking at the package dependencies in `@spinnaker/titus` and took a guess of what should be provided by `deck`.